examples/crowdfunding/scripts/createApp.js

```diff
@@ -0,0 +1,95 @@
+const {
+  executeTransaction,
+  TransactionType,
+  SignType,
+  update,
+  intToBytes,
+  addressToBytes
+} = require('algob');
+
+async function run (runtimeEnv, deployer) {
+  const masterAccount = deployer.accountsByName.get('master-account');
+  const creatorAccount = deployer.accountsByName.get('alice');
+  const donorAccount = deployer.accountsByName.get('john');
+
+  const algoTxnParams = {
+    type: TransactionType.TransferAlgo,
+    sign: SignType.SecretKey,
+    fromAccount: masterAccount,
+    toAccountAddr: creatorAccount.addr,
+    amountMicroAlgos: 200000000,
+    payFlags: {}
+  };
+  await executeTransaction(deployer, algoTxnParams);
+
+  // Get begin date timestamp to pass in
+  const beginDateTimestamp = new Date();
```

Comment:
Let's remove `Timestamp` suffix. Not needed, and wrong - the value here is a `Date` object.
Same below.

Reply:
ok